app/presenters/setting_presenter.rb

```diff
@@ -55,7 +55,7 @@ def settings_type
   # ----- UI helpers ------
 
   def category_label
-    category.constantize.humanized_category || _(category_name)
+    _(SettingManager.categories[category]) || category.constantize.humanized_category || _(category_name)
```

Comment:
we will probably need to check for not passing `_(nil)`, as there might be dragons.

Reply:
Well, I am pretty sure no language would define nil but I like your thinking!

```
[1] pry(main)> _(nil)
=> nil
```

Reply:
Thanks, I'll add a test case for this then and be done with it :+1:

Reply:
Eh, it was wrong anyway, we pass it to `_()` in the view :shrug: